index.html

```diff
@@ -98,22 +99,21 @@ <h3 class="program_title">Network</h3>
 
     <footer id="partners">
         <div class="partners_container">
-            <h2 class="progam_section_title">Partners</h2>
+            <h2 class="progam_section_title">Partner</h2>
             <hr class="program_line">
             <div class="partners">
                 <img class="partner_one pr" src="./assets/partners/partner_one.png"></img>
                 <img class="partner_three pr" src="./assets/partners/partner_three.png"></img>
                 <img class="partner_six pr" src="./assets/partners/partner_six.png"></img>
                 <img class="partner_two pr" src="./assets/partners/partner_two.png"></img>
                 <img class="partner_four pr" src="./assets/partners/partner_four.png"></img>
```

Comment:
- [ ] Kindly for screenreaders and accessibility needs add an `alt` tag to each `img` of your project. In this way, the user will know what's going on if he/she is using a screen reader or the img isn't loaded in some browser 🥇